docs/Installation.md

```diff
@@ -87,7 +87,7 @@ please follow the [advanced installation instructions](#advanced-local-installat
 #### Advanced: Local Installation for Development
 
 You can [add the local](https://docs.unity3d.com/Manual/upm-ui-local.html)
-`com.unity.ml-agents` package (from the repository that you just cloned) to our
+`com.unity.ml-agents` package (from the repository that you just cloned) to your
```

Comment:
Good catch, can you make sure that gets made on master too?

Reply:
I fixed this on master in my PR. I think this is from the cherry-pick of that commit.